chia/wallet/wallet_transaction_store.py

```diff
@@ -286,6 +287,11 @@ async def get_transactions_between(
         else:
             query_str = SortKey[sort_key].ascending()
 
+        if confirmed is None:
+            confirmed_str = ""
+        else:
+            confirmed_str = "AND confirmed=" + str(int(confirmed))
+
```

Comment:
```suggestion
        confirmed_str = ""
        if confirmed is not None:
            confirmed_str = f"AND confirmed={int(confirmed)}"
         
```

nit, feel free to just close this :)